bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -311,12 +320,17 @@ public void dispose() {
      */
     @Override
     public synchronized void restartWebsockets() {
-        logger.debug("Restarting GardenaSmart Webservice");
+        logger.debug("Restarting GardenaSmart Webservices");
         stopWebsockets();
         try {
             startWebsockets();
         } catch (Exception ex) {
-            logger.warn("Restarting GardenaSmart Webservice failed: {}, restarting binding", ex.getMessage());
+            // restart binding
+            if (logger.isDebugEnabled()) {
```

Comment:
What's the point in this logic - why degrade log level if debug logging is enabled?

Reply:
I just wanted to log the whole stacktrace, when debug logging is enabled. Otherwise only the message of the exception is logged.

Reply:
OK, fair enough. Why not keep the log level the same in both cases though?

Reply:
You are right. I think it makes more sense to use `logger.warn` for both cases. I will change that.